dist/resources/items.js

```diff
@@ -53,7 +53,8 @@ module.exports = function (api) {
     create: function create() {
       var params = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : {};
       var callback = arguments[1];
-      var amount = params.amount,
+      var name = params.name,
+          amount = params.amount,
```

Comment:
@ankitdas13 make there are consistency in the variable declaration if your using `var` for variable declaration then use it everywhere in a file where its been declared

Reply:
Also see proper indentation are followed